From cbb77f161d60964733f332e4ccfb9f240947c3ef Mon Sep 17 00:00:00 2001
From: root <root@fit-build-116.mtl.labs.mlnx>
Date: Tue, 5 Apr 2022 21:35:55 +0300
Subject: [PATCH backport 5.10 168/182] TMP: mlxsw: minimal: Ignore error
 reading SPAD register

WA until FW will add support for SPAD register for all systems.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>
---
 drivers/net/ethernet/mellanox/mlxsw/minimal.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/mellanox/mlxsw/minimal.c b/drivers/net/ethernet/mellanox/mlxsw/minimal.c
index 0b605c6aa637..5fd319697c94 100644
--- a/drivers/net/ethernet/mellanox/mlxsw/minimal.c
+++ b/drivers/net/ethernet/mellanox/mlxsw/minimal.c
@@ -64,11 +64,12 @@ static int mlxsw_m_base_mac_get(struct mlxsw_m *mlxsw_m)
 {
 	char spad_pl[MLXSW_REG_SPAD_LEN] = {0};
 	int err;
-
+#if 0
 	err = mlxsw_reg_query(mlxsw_m->core, MLXSW_REG(spad), spad_pl);
 	if (err)
 		return err;
 	mlxsw_reg_spad_base_mac_memcpy_from(spad_pl, mlxsw_m->base_mac);
+#endif
 	return 0;
 }
 
-- 
2.20.1

